examples/ActivityLogger/Program.cs

```diff
@@ -11,6 +11,7 @@ internal class Program
     {
         public static void Main(string[] args)
         {
+            Environment.SetEnvironmentVariable("App_Environment", "Development");
```

Comment:
Why is this set here?

Reply:
Are you asking why this is set in this particular spot because there is a better place for it? Or why is it set at all? 
For the second one, I think it is needed to make sure we get the right app settings.

Reply:
This is the example project, a simple console app.  When I tried to run it, it blew up because the environment variable was not set and it was expected by `CreateHost()`.  I didn't know the proper way to set it, but I figured this would be good enough for an example project.  Another solution could be to modify `CreateHost()` to not depend on the environment variable.